cmd/support-scheduler/res/docker/configuration.toml

```diff
@@ -59,9 +59,3 @@ File = '/edgex/logs/edgex-support-scheduler.log'
     Path = 'api/v1/event/removeold/age/604800000'
```

Comment:
Just realized its the same for this line
```suggestion
    Path = '/api/v1/event/removeold/age/604800000'
```

Reply:
You are right, fixed too

Reply:
Is this fixed now?
If so having this in the Delhi would be fantastic